static/js/components/analytics/AnalyticsChart.js

```diff
@@ -231,6 +233,31 @@ export class AnalyticsChart extends React.Component {
     }
     return viewsAtTimesByChannel
   }
+
+  _getYTicks(opts) {
+    const { analyticsData } = opts
+    const sortedTotalViewsValues = analyticsData.times
+      .map(time => {
+        const viewsAtTime = analyticsData.views_at_times[time]
+        return _.sum(
+          analyticsData.channels.map(channel => {
+            return viewsAtTime[channel] || 0
+          })
+        )
+      })
+      .sort()
+    const ticks = [0]
+    if (sortedTotalViewsValues.length > 0) {
+      if (sortedTotalViewsValues.length > 1) {
+        ticks.push(Math.round(_.mean(sortedTotalViewsValues)))
```

Comment:
Just to be sure, is the expected y tick values always supposed to be minimum, mean, maximum?

Reply:
Yes, that is the expectation.